mitmproxy/mitmproxy/proxy/config.py

```diff
@@ -181,7 +182,9 @@ def process_proxy_options(parser, options):
         certs.append(parts)
 
     if options.http2 and not tcp.HAS_ALPN:
-        raise RuntimeError("HTTP2 support requires OpenSSL 1.0.2 or above.")
```

Comment:
How does that play out with mitmproxy? Users won't be able to see this, will they?